content/en/blog/_posts/2022-08-14-pod-has-network-condition.md

```diff
@@ -0,0 +1,109 @@
+---
+layout: blog
+title: 'Kubernetes 1.25: PodHasNetwork condition for pods'
+date: 2022-05-27
+slug: pod-has-network-condition
+---
+
+**Author:**
+Deep Debroy (Apple)
+
+Kubernetes 1.24 introduces Alpha support for a new kubelet managed pod condition
```

Comment:
similar to previous comment, v1.24 would be more appropriate

Reply:
good catch - fixed!